### ISSUE TITLE
convert darknet53 network weights

### ISSUE THREAD

**Issue by JeremBlain** [qqwweee/keras-yolo3#243]
Hello, I'd like to convert darknet53.
Am I right that convert.py don't convert last convolutionnal layer with 1000 outputs ? (the one before the softmax)
in the darknet53.cfg in this repo, there is no such layer at the end.
If I try to convert the .cfg from darknet itself, it tells me that it didn't know avgpool layer (this layer is just before the last convolutionnal layer)
I need the weight of that layer, how could I get them ? (I don't want to retrain the last layer)

Thanks.

**Comment by AlbertoSabater** [comment]
Hi, I have the same problem. Did you manage to solve it?

**Comment by bluesy7585** [comment]
i think darknet53 is used as feature extractor in yolov3.
last 2 layers is removed and more layers are attached to become yolov3.
avgpool and softmax 1000 are not part of yolo network.

**Comment by AlbertoSabater** [comment]
I finally solved it. Those are the layers related to the feature extractor. I finally could convert those weights to a .h5 file with the .cfg file located in the repo. I was using a different one

**Comment by wenredo** [comment]
So what did you do? Removing the two last layers from the cfg file? Thanks in advance

**Comment by AlbertoSabater** [comment]
No, I didn't change anything, just using [this](https://github.com/pjreddie/darknet/blob/master/cfg/darknet53.cfg) .cfg file.

**Comment by wenredo** [comment]
To train would you just set the yolov3 anchors then?

**Comment by AlbertoSabater** [comment]
Yes
